src/components/strategies/useUpdateStrategy.ts

```diff
@@ -7,19 +7,24 @@ import {
   useQueryClient,
   useUpdateStrategyQuery,
 } from 'libs/queries';
-import { PathNames, useNavigate } from 'libs/routing';
+import { useNavigate } from 'libs/routing';
 import { useWeb3 } from 'libs/web3';
+import { useState } from 'react';
+import { ONE_AND_A_HALF_SECONDS_IN_MS } from 'utils/time';
+import { handleTxStatusAndRedirectToOverview } from './create/utils';
 
 export const useUpdateStrategy = () => {
```

Comment:
I really think that we should split all these edits into their own components and hooks. This is destined to become a big pile of spaghetti

Reply:
Added this issue to deeplink issue